features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql

```diff
@@ -2047,25 +2047,37 @@ CREATE TABLE AM_KEY_MANAGER (
   );
 
 IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[AM_GW_PUBLISHED_API_DETAILS]') AND TYPE IN (N'U'))
-CREATE TABLE IF NOT EXISTS AM_GW_PUBLISHED_API_DETAILS (
+CREATE TABLE  AM_GW_PUBLISHED_API_DETAILS (
   API_ID varchar(255) NOT NULL,
   TENANT_DOMAIN varchar(255),
-  API_PROVIDER varchar(255),
+  API_PROVIDEta),
```

Comment:
shall we fix this?

Reply:
https://github.com/wso2/carbon-apimgt/pull/8929/commits/0b82c3da97e20e9617deed4295b2208523f377e4